src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/CollectSVEvidence.java

```diff
@@ -58,21 +63,33 @@
  *     <li>sample name</li>
  * </ul>
  *
- * The third output file, which should be named "*.ac.txt" or "*.ac.txt.gz" specifies allele counts:
- * For each locus specified in an input VCF as a simple SNP, it gives a count, for each base, of the
- * number of reads that cover that locus.
+ * The third output file, which should be named "*.sd.txt" or "*.sd.txt.gz" specifies site depth counts:
+ * For each locus specified in an input VCF as a simple, biallelic SNP, it gives a count, for each
+ * base call, of the number of reads that cover that locus.
  * It has the following columns:
  *
  * <ul>
  *     <li>contig</li>
  *     <li>position</li>
- *     <li>ref allele</li>
+ *     <li>sampleName</li>
  *     <li>A observations</li>
  *     <li>C observations</li>
  *     <li>G observations</li>
  *     <li>T observations</li>
  * </ul>
  *
+ * The fourth output file, which should be named "*.rd.txt" or "*.rd.txt.gz" specifies read depths:
+ * For each interval specified by a 3-column, tab delimited input file, the number of reads that
+ * start in that interval are reported.
+ * It has the following columns:
+ *
+ * <ul>
+ *     <li>contig</li>
+ *     <li>starting position</li>
+ *     <li>ending position</li>
+ *     <li>read count</li>
+ * </ul>
+ *
```

Comment:
```suggestion
 *
 * Note when only collecting RD evidence, users should provide the same interval list with -L as --depth-evidence-intervals in order to avoid processing unused reads outside the intervals.
```